Makefile

```diff
@@ -1,4 +1,5 @@
 BINDIR      := $(CURDIR)/bin
+HELM_INSTALL_DIR := /usr/local/bin
```

Comment:
If we want this variable to be able to be set in the environment it should use `?=` to set the value. This way it could be set or changed in the outside environment. Like the `BINNAME` variable in this file.